package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@hackolade/fetch",
+  "version": "1.0.0",
+  "description": "A HTTP client that works in the browser and in Electron",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/esm/index.mjs",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.mjs",
+      "require": "./dist/cjs/index.cjs"
+    }
+  },
+  "scripts": {
+    "build": "swc -C module.type=es6 ./src -o ./dist/esm/index.mjs && swc -C module.type=commonjs ./src -o ./dist/cjs/index.cjs",
+    "test": "npm run build && node --test ./test/*.spec.js",
+    "test:app:electron": "electron ./test/resources/electron-app/main.js"
+  },
+  "peerDependencies": {
+    "electron": ">=31"
+  },
+  "devDependencies": {
+    "@swc/cli": "^0.4.0",
```

Comment:
could you please lock package versions? We don't want any _surprises_

Reply:
Done

Reply:
@thomas-jakemeyn @chulanovskyi-bs while useful with a regular update process for `production` dependencies, here it actually doesn't really bring any value (almost the contrary for dev dependencies where we won't get semantic updates).   On studio side it's important because a lot of dev deps are actually production ones but here tbh i don't see the requirement.  Having a hook that checks if packages updates are available e.g. something like `npx npm-check-updates` would be wishable.